Mathlib/Data/Int/Cast/Basic.lean

```diff
@@ -61,30 +61,30 @@ theorem cast_zero : ((0 : ℤ) : R) = 0 :=
 #align int.cast_zero Int.cast_zeroₓ
 -- type had `HasLiftT`
 
-@[simp high, nolint simpNF, norm_cast] -- this lemma competes with `Int.ofNat_eq_cast` to come later
-theorem cast_ofNat (n : ℕ) : ((n : ℤ) : R) = n :=
+@[simp, norm_cast]
+theorem cast_natCast (n : ℕ) : ((n : ℤ) : R) = n :=
   AddGroupWithOne.intCast_ofNat _
-#align int.cast_coe_nat Int.cast_ofNatₓ
+#align int.cast_coe_nat Int.cast_natCastₓ
 -- expected `n` to be implicit, and `HasLiftT`
-#align int.cast_of_nat Int.cast_ofNatₓ
+#align int.cast_of_nat Int.cast_natCastₓ
```

Comment:
Should that be
```suggestion
#align int.cast_of_nat Int.cast_ofNat
```
(maybe with an `ₓ`?

Reply:
No, this is correct. The `cast_of_nat` lemma in lean 3 is morally the same as `cast_natCast`; both are about coercions not literals.

The lean3 version of `cast_ofNat` is `cast_bit0` / `cast_bit1`.